EasyDisenchant.lua

```diff
@@ -14,9 +14,43 @@ local _M = {
 	buttonRenderingCache = {},
 };
 
+EasyDisenchantBlacklist = {};
```

Comment:
Variables defined by the SavedVariables tag should be initiated after the `ADDON_LOADED` event has fired for the add-on, not during script parse.